urllib3/connectionpool.py

```diff
@@ -41,11 +40,8 @@
 from .util.retry import Retry
 from .util.timeout import Timeout
 from .util.url import get_host, Url
-
-
-if six.PY2:
-    # Queue is imported for side effects on MS Windows
-    import Queue as _unused_module_Queue  # noqa: F401
+from .util.queue import LifoQueue
+from .packages.six.moves import queue
```

Comment:
Why did this move down here? The rest of the imports from `packages` live elsewhere?

Reply:
No reason, I deleted it and upon realizing that `queue.Empty` was used I put it back but not in the right place. I'll fix it and amend the commit.